# Description: Fixes shef crashing on startup about resources not found.
# Fixed upstream for next Chef version release.
Index: chef-0.9.4+dfsg/chef/lib/chef/shef/ext.rb
===================================================================
--- chef-0.9.4+dfsg.orig/chef/lib/chef/shef/ext.rb	2010-07-02 10:40:24.000000000 -0600
+++ chef-0.9.4+dfsg/chef/lib/chef/shef/ext.rb	2010-07-02 10:40:57.000000000 -0600
@@ -15,6 +15,11 @@
 # limitations under the License.
 #
 
+require 'chef/recipe'
+require 'fileutils'
+require 'chef/version'
+require 'chef/shef/shef_session'
+
 module Shef
   module Extensions
     
